pkg/apis/core/v1alpha1/types_shoot.go

```diff
@@ -688,6 +688,20 @@ type KubeAPIServerConfig struct {
 	// Defaults to 1h.
 	// +optional
 	EventTTL *metav1.Duration `json:"eventTTL,omitempty" protobuf:"bytes,12,opt,name=eventTTL"`
+	// Logging contains configuration for the logs level and http access logs
+	// +optional
+	Logging *Logging `json:"logging,omitempty" protobuf:"bytes,13,opt,name=logging"`
+}
+
+// Logging contains configuration for the logs level and http access logs
+type Logging struct {
+	// Verbosity is the kube-apiserver log verbosity level
+	// Defaults to 2.
+	// +optional
+	Verbosity *int32 `json:"verbosity,omitempty" protobuf:"bytes,1,name=verbosity"`
```

Comment:
This is an optional field, should it be like this and the same for other fields?
```suggestion
	Verbosity *int32 `json:"verbosity,omitempty" protobuf:"bytes,1,opt,name=verbosity"`
```

Reply:
Isn't this stuff auto-generated? 👀 cc @ialidzhikov

Reply:
I removed the protobuf tag from the field, then ran `hack/update-protobuf.sh`.

The generated tag is:
```go
	Verbosity *int32 `json:"verbosity,omitempty" protobuf:"varint,1,opt,name=verbosity"`
```

Even the type in the current revision is wrong, it is set to `bytes` but the generator generates `varint`. I assume that @vlvasilev manually set the protobuf tag and it seems that generator does not overwrite the tag when it is already specified.

Reply:
Thanks @ialidzhikov. The PR looks good except for this thing. @vlvasilev please check and regenerate.